packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -800,12 +800,10 @@ class RenderListWheelViewport
   /// Paints all children visible in the current viewport.
   void _paintVisibleChildren(PaintingContext context, Offset offset) {
     RenderBox? childToPaint = firstChild;
-    ListWheelParentData? childParentData = childToPaint?.parentData as ListWheelParentData?;
-
-    while (childParentData != null) {
-      _paintTransformedChild(childToPaint!, context, offset, childParentData.offset);
+    while (childToPaint != null) {
+      final ListWheelParentData childParentData = childToPaint.parentData! as ListWheelParentData;
```

Comment:
It may change the previous behaviour if `parentData` is null. I guest it can not be null?

Reply:
if you're painting then you must have parentData, yeah.